jhub_apps/configuration.py

```diff
@@ -54,12 +55,13 @@ def install_jhub_apps(c, spawner_to_subclass):
             },
             {
                 "name": "launcher",
-                "url": "http://127.0.0.1:5000",
+                "url": "http://0.0.0.0:5000",
                 "command": [
                     c.JAppsConfig.python_exec,
                     "-m",
                     "jhub_apps.launcher.main",
-                    f"--origin-host={get_origin_host(c.JupyterHub.bind_url)}",
+                    # f"--origin-host={get_origin_host(c.JupyterHub.bind_url)}",
+                    f"--origin-host=*",
```

Comment:
Although, I believe this is a security risk, any reason for this change?

Reply:
We have our jupyterhub behind a VPN so this wasn't an issue

I can see how this would be important for other users, so I added the option for this to be configurable